src/parameterizations/vertical/MOM_bkgnd_mixing.F90

```diff
@@ -159,10 +179,17 @@ subroutine bkgnd_mixing_init(Time, G, GV, param_file, diag, CS)
 
   call get_param(param_file, mdl, 'DEBUG', CS%debug, default=.False., do_not_log=.True.)
 
+  prandtl_bkgnd_default = Kv/CS%Kd
```

Comment:
I think this line is why this PR is failing tests:
- This is yielding NaN or Infinity depending on the compiler and configuration.
- PRANDTL_BKGND is also being logged in the output in a format that cannot be read back in. 

I suspect the resolution is to only read/log PRANDTL_BKGND in those modes where you are actually using it (when it is not infinite).

Perhaps we should discuss this parameter in the dev call?